src/selectors/getAllProjectKeys.js

```diff
@@ -3,8 +3,15 @@ import {createSelector} from 'reselect';
 export default createSelector(
   state => state.get('projects'),
   projects => projects.
-    filter(project => project.updatedAt).
-    sortBy(project => -project.updatedAt).
+    sort((firstProj, secondProj) => {
+      if (firstProj.updatedAt === null) {
+        return -1;
```

Comment:
Technically should be 0 if they are _both_ null right?

Reply:
There shouldn't be a situation with 2 pristine projects

Reply:
That’s correct now, of course, but do we want to rely on that always being true? Right now this sorting algorithm would produce non-deterministic sorting results if there ever were a situation where there could be two projects with null `updatedAt`s. Seems worth the fairly minimal additional code to handle that case?